packages/wiz-ui-react/src/components/base/show-more-less/components/show-more-less.tsx

```diff
@@ -74,23 +62,25 @@ const ShowMoreLess: FC<Props> = ({
         >
           {children}
         </div>
-        <div className={showMoreLessSummaryStyle} onClick={handleClick}>
-          <div
-            className={clsx(
-              showMoreLessMessageStyle,
-              bgColor && backgroundStyle[bgColor],
-              colorStyle[fontColor]
-            )}
-          >
-            <WizHStack align="center" justify="between" gap="xs2">
+        <div
+          className={showMoreLessSummaryStyle[variant]}
+          onClick={handleClick}
+        >
+          <div className={clsx(showMoreLessMessageStyle)}>
+            <WizHStack
+              align="center"
+              justify="between"
+              gap="xs2"
+              height={variant === "mobile" ? "2.75rem" : "1.375rem"}
+            >
               <div>{isOpen ? closeMessage : openMessage}</div>
               <div
                 className={clsx(
                   showMoreLessExpandIconStyle,
                   isOpen && showMoreLessRotateIconStyle
                 )}
               >
-                <WizIcon size="xl2" icon={WizIExpandMore} color={fontColor} />
+                <WizIcon size="xl2" icon={WizIExpandMore} color={"inherit"} />
```

Comment:
[nits] 定数で渡すなら中括弧は不要です。

```suggestion
                <WizIcon size="xl2" icon={WizIExpandMore} color="inherit" />
```